src/markdownit/splitMixedLists.js

```diff
@@ -53,6 +53,14 @@ export default function splitMixedLists(md) {
 	})
 }
 
+/**
+ * @param {object} token MarkdownIT token
+ * @param {string} cls Class name to query
+ */
+function includesClass(token, cls) {
+	return token.attrGet('class')?.split(' ').includes(cls) || false
```

Comment:
Not realy as that would match if the queried class name is part of another class, consider this: 
```js
token = {
  class: "foo-bar left"
}

includesClass(token, "bar")
```

without the `split` this would return `true` even if `bar` is not a class of that token.